paths.yaml

```diff
@@ -507,6 +507,9 @@
 "/projects/{project_id}/orders/{id}/confirm":
   patch:
     "$ref": "./paths/orders/confirm.yaml"
+"/projects/{id}/quality_performance_score":
+  post:
+    "$ref": "./paths/quality_performance_score/index.yaml"
```

Comment:
somehow `post` and `index` don't go together. why is this a POST to begin with?

Reply:
The request query would be too long if it was in the URL. We want to support 100 ids at a time.